jekyll/_cci2/contexts.md

```diff
@@ -48,8 +48,51 @@ workflows:
 
 To rerun a job and use the context, it **must** be rerun from the Workflows page of the CircleCI application.
 
+## Securing a Context
+
+CircleCI enables you to secure secrets at run time by adding security groups to contexts. Only Org Admins may add *security groups* to a new or existing context. Security groups are definied as LDAP groups or GitHub teams. After the security group is added to a context, only members of that security group may access or use the environment variables of the secure context. 
+
+The default security group is `organization` and enables any member of the org to use the context.
+
+## Runnning Workflows with a Secure Context
+
+To invoke a workflow that uses a secure context, a user must be a member of one of the security groups for the context or the workflow will fail with the status of `Unauthorized`. If you add a context to your workflow and you are **not** a member of any of the context groups, the job will fail as `Unauthorized`.
```

Comment:
>the job will fail as `Unauthorized`
the workflow will fail as `Unauthorized`.

Reply:
thanks, fixed